addons/web/controllers/export.py

```diff
@@ -378,7 +378,9 @@ def get_fields(self, model, domain, prefix='', parent_name='',
 
         fields['id']['string'] = request.env._('External ID')
 
-        if parent_field:
+        if Model._table_query is not None:
```

Comment:
Check `Model._is_an_ordinary_table()` instead to match exactly the reason why it will fail afterwards

Reply:
Indeed :+1: